packages/web/src/common/components/Table.tsx

```diff
@@ -13,11 +13,17 @@ interface TableProps<T> {
   footer?: React.ReactNode;
 }
 
+const TableInnerWrapper = styled.div`
+  width: calc(100% + (100vw - 100%));
+  padding: 0 calc((100vw - 100%) / 2);
+  overflow-x: auto;
+`;
+
 const TableInner = styled.table<{ height?: number }>`
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  width: 100%;
+  min-width: 100%;
```

Comment:
그리고 요걸 추가해주면 width를 의도적으로 키워주지 않아도 되는 것 같긴 한데.. 확인해봐줄 수 있어?

```suggestion
  min-width: 100%;
  width: fit-content;
```